app/models/user.rb

```diff
@@ -18,6 +18,10 @@ def three_most_recent_posts
 
   # PostsCounter must be an integer greater than or equal to zero.
   validates :posts_counter, numericality: { allow_nil: true, only_integer: true, greater_than_or_equal_to: 0 }
+
+  def admin?
+    role == 'admin'
+  end
```

Comment:
- Kudos for creating the custom `admin?` method in the User model. 💯

Reply:
Thank you @MahmoudBakr23